pages/api/performances/collect.js

```diff
@@ -102,3 +103,13 @@ export const getPerformances = async filter => {
     }
   );
 };
+
+const getSetting = async id => {
+  let setting = await prisma.setting.findUnique({
+    where: {
+      id: parseInt(id),
+    },
+  });
+  if (!setting) return {};
```

Comment:
Did you test when it doesn't find a setting with the id?

Reply:
Don't anticipate this happening but I added a check. Thanks for catching this!